### PR TITLE
CSS and Material Updates / Improvements. Change Reddit Link to Docker

### PR DESCRIPTION
**Pull request by ellnic**
CSS Improvements to default theme
Complete Light Theme Redesign
Material Icon update from v2 to v4.6.95 (to add mdi-docker)
Change the Reddit link in the Navbar to a link to the Docker repo
Removal of all !important tags in all themes

### CODE REVIEW

File: static/css/themes/Slate.css [hunk 1]
```diff
@@ -3,4 +3,7938 @@
...
...-dark .navbar-nav .show>.nav-link,.navbar-dark .navbar-nav .active>.nav-link,.navbar-dark .navbar-nav .nav-link.show,.navbar-dark .navbar-nav .nav-link.active{color:#fff}.navbar-dark .navbar-toggler{color:rgba(255,255,255,0.5);border-color:rgba(255,255,255,0.1)}.navbar-dark .navbar-toggler-icon{background-image:url("data:image/svg+xml;charset=utf8,%3Csvg viewBox='0 0 30 30' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath stroke='rgba(255,255,255,0.5)' stroke-width='2' stroke-linecap='round' stroke-miterlimit='10' d='M4 7h22M4 15h22M4 23h22'/%3E%3C/svg%3E")}.navbar-dark .navbar-text{color:rgba(255,255,255,0.5)}.navbar-dark .navbar-text a{color:#fff}.navbar-dark .navbar-text a:hover,.navbar-dark .navbar-text a:focus{color:#fff}.card{position:relative;display:flex;flex-direction:column;min-width:0;word-wrap:break-word;background-color:#3A3F44;background-clip:border-box;border:1px solid rgba(0,0,0,0.125);border-radius:.25rem}.card>hr{margin-right:0;margin-left:0}.card>.list-group:first-child .list-group-item:first-child{border-top-left-radius:.25rem;border-top-right-radius:.25rem}.card>.list-group:last-child .list-group-item:last-child{border-bottom-right-radius:.25rem;border-bottom-left-
```
Comment (barbequesauce) [inline static/css/themes/Slate.css]: Codacy Issue found: [Expected RBRACE at line 32, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4606161)

**Comment by barbequesauce** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 25
- Added 130
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/Watcher3/Watcher3/pullRequest?prid=4606161&bid=15455368)

File: static/css/themes/Midnight.css [hunk 2]
```diff
@@ -3,4 +3,7938 @@
...
...how>.nav-link,.navbar-dark .navbar-nav .active>.nav-link,.navbar-dark .navbar-nav .nav-link.show,.navbar-dark .navbar-nav .nav-link.active{color:#fff}.navbar-dark .navbar-toggler{color:rgba(255,255,255,0.5);border-color:rgba(255,255,255,0.1)}.navbar-dark .navbar-toggler-icon{background-image:url("data:image/svg+xml;charset=utf8,%3Csvg viewBox='0 0 30 30' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath stroke='rgba(255,255,255,0.5)' stroke-width='2' stroke-linecap='round' stroke-miterlimit='10' d='M4 7h22M4 15h22M4 23h22'/%3E%3C/svg%3E")}.navbar-dark .navbar-text{color:rgba(255,255,255,0.5)}.navbar-dark .navbar-text a{color:#fff}.navbar-dark .navbar-text a:hover,.navbar-dark .navbar-text a:focus{color:#fff}.card{position:relative;display:flex;flex-direction:column;min-width:0;word-wrap:break-word;background-color:#011f3d;background-clip:border-box;border:1px solid rgba(0,0,0,0.125);border-radius:.25rem}.card>hr{margin-right:0;margin-left:0}.card>.list-group:first-child .list-group-item:first-child{border-top-left-radius:.25rem;border-top-right-radius:.25rem}.card>.list-group:last-child .list-group-item:last-child{border-bottom-right-radius:.25rem;border-bottom-left-radius:.25rem}.card-
```
Comment (barbequesauce) [inline static/css/themes/Midnight.css]: Codacy Issue found: [Expected RBRACE at line 25, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4606161)

File: static/css/themes/Slate.css (hunk 1 above)
Comment (barbequesauce) [inline static/css/themes/Slate.css]: Codacy Issue found: [Expected RBRACE at line 17, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4606161)

Comment (barbequesauce) [inline static/css/themes/Slate.css]: Codacy Issue found: [Expected RBRACE at line 11, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4606161)

File: static/css/themes/Light.css [hunk 3]
```diff
@@ -3,4 +3,8015 @@
...
...wrap:wrap;padding:.75rem 1rem;margin-bottom:1rem;list-style:none;background-color:#e9ecef}.breadcrumb-item+.breadcrumb-item::before{display:inline-block;padding-right:.5rem;padding-left:.5rem;color:#868e96;content:"/"}.breadcrumb-item+.breadcrumb-item:hover::before{text-decoration:underline}.breadcrumb-item+.breadcrumb-item:hover::before{text-decoration:none}.breadcrumb-item.active{color:#868e96}.pagination{display:flex;padding-left:0;list-style:none}.page-link{position:relative;display:block;padding:.5rem .75rem;margin-left:-1px;line-height:1.25;color:#2196F3;background-color:#fff;border:1px solid #dee2e6}.page-link:hover{color:#0a6ebd;text-decoration:none;background-color:#e9ecef;border-color:#dee2e6}.page-link:focus{z-index:2;outline:0;box-shadow:0 0 0 .2rem rgba(33,150,243,0.25)}.page-link:not(:disabled):not(.disabled){cursor:pointer}.page-item:first-child .page-link{margin-left:0}.page-item.active .page-link{z-index:1;color:#fff;background-color:#2196F3;border-color:#2196F3}.page-item.disabled .page-link{color:#868e96;pointer-events:none;cursor:auto;background-color:#fff;border-color:#dee2e6}.pagination-lg .page-link{padding:.75rem 1.5rem;font-size:1.17188rem;line-height:1.5}.
```
Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Expected COLON at line 4101, col 1.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4606161)

Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Expected RBRACE at line 18, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4606161)

Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Expected RBRACE at line 15, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4606161)

File: static/css/themes/Spruce.css [hunk 4]
```diff
@@ -3,4 +3,7938 @@
...
...v-link,.navbar-dark .navbar-nav .active>.nav-link,.navbar-dark .navbar-nav .nav-link.show,.navbar-dark .navbar-nav .nav-link.active{color:#fff}.navbar-dark .navbar-toggler{color:rgba(255,255,255,0.5);border-color:rgba(255,255,255,0.1)}.navbar-dark .navbar-toggler-icon{background-image:url("data:image/svg+xml;charset=utf8,%3Csvg viewBox='0 0 30 30' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath stroke='rgba(255,255,255,0.5)' stroke-width='2' stroke-linecap='round' stroke-miterlimit='10' d='M4 7h22M4 15h22M4 23h22'/%3E%3C/svg%3E")}.navbar-dark .navbar-text{color:rgba(255,255,255,0.5)}.navbar-dark .navbar-text a{color:#fff}.navbar-dark .navbar-text a:hover,.navbar-dark .navbar-text a:focus{color:#fff}.card{position:relative;display:flex;flex-direction:column;min-width:0;word-wrap:break-word;background-color:#052c25;background-clip:border-box;border:1px solid rgba(0,0,0,0.125);border-radius:.25rem}.card>hr{margin-right:0;margin-left:0}.card>.list-group:first-child .list-group-item:first-child{border-top-left-radius:.25rem;border-top-right-radius:.25rem}.card>.list-group:last-child .list-group-item:last-child{border-bottom-right-radius:.25rem;border-bottom-left-radius:.25rem}.card-body{fl
```
Comment (barbequesauce) [inline static/css/themes/Spruce.css]: Codacy Issue found: [Expected RBRACE at line 33, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4606161)

File: static/css/themes/Slate.css (hunk 1 above)
Comment (barbequesauce) [inline static/css/themes/Slate.css]: Codacy Issue found: [Expected RBRACE at line 35, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4606161)

File: static/css/themes/Spruce.css (hunk 4 above)
Comment (barbequesauce) [inline static/css/themes/Spruce.css]: Codacy Issue found: [Expected RBRACE at line 14, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4606161)

File: static/css/themes/Midnight.css (hunk 2 above)
Comment (barbequesauce) [inline static/css/themes/Midnight.css]: Codacy Issue found: [Expected RBRACE at line 34, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4606161)

File: static/css/themes/Spruce.css (hunk 4 above)
Comment (barbequesauce) [inline static/css/themes/Spruce.css]: Codacy Issue found: [Expected RBRACE at line 18, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4606161)

File: static/css/themes/Slate.css (hunk 1 above)
Comment (barbequesauce) [inline static/css/themes/Slate.css]: Codacy Issue found: [Expected RBRACE at line 36, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4606161)

File: static/css/themes/Light.css (hunk 3 above)
Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Expected RBRACE at line 28, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4606161)

File: static/css/themes/Slate.css (hunk 1 above)
Comment (barbequesauce) [inline static/css/themes/Slate.css]: Codacy Issue found: [Expected IDENT at line 5062, col 23.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4606161)

File: static/css/themes/Spruce.css (hunk 4 above)
Comment (barbequesauce) [inline static/css/themes/Spruce.css]: Codacy Issue found: [Expected RBRACE at line 34, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4606161)

File: static/css/themes/Slate.css (hunk 1 above)
Comment (barbequesauce) [inline static/css/themes/Slate.css]: Codacy Issue found: [Expected RBRACE at line 33, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4606161)

Comment (barbequesauce) [inline static/css/themes/Slate.css]: Codacy Issue found: [Expected RBRACE at line 26, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4606161)

File: static/css/themes/Default.css [hunk 5]
```diff
@@ -8,8 +8,8 @@
 :root {
 	--watcher-light: #F7931E;
 	--watcher-mid: #F5861E;
-	--watcher-dark: #F05923
-	--blue:#2196F3;
+	--watcher-dark: #F05923;
+	--blue: #2196F3;
```
Comment (barbequesauce) [inline static/css/themes/Default.css]: Codacy Issue found: [Expected RBRACE at line 12, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4606161)

File: static/css/themes/Spruce.css (hunk 4 above)
Comment (barbequesauce) [inline static/css/themes/Spruce.css]: Codacy Issue found: [Expected RBRACE at line 26, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4606161)

File: static/css/themes/Midnight.css (hunk 2 above)
Comment (barbequesauce) [inline static/css/themes/Midnight.css]: Codacy Issue found: [Expected RBRACE at line 20, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4606161)

File: static/css/themes/Slate.css (hunk 1 above)
Comment (barbequesauce) [inline static/css/themes/Slate.css]: Codacy Issue found: [Expected RBRACE at line 28, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4606161)

Comment (barbequesauce) [inline static/css/themes/Slate.css]: Codacy Issue found: [Expected RBRACE at line 31, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4606161)

File: static/css/themes/Midnight.css (hunk 2 above)
Comment (barbequesauce) [inline static/css/themes/Midnight.css]: Codacy Issue found: [Expected RBRACE at line 29, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4606161)

Comment (barbequesauce) [inline static/css/themes/Midnight.css]: Codacy Issue found: [Expected RBRACE at line 22, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4606161)

**Comment by ellnic** [comment]
I am not sure why Codacy is moaning about the brackets in the root definition here. I have tripple checked it and every site I have looked at says they are correctly defined. I don't see an issue.

**Comment by barbequesauce** [comment]
Dont stress over it - I'll get around to cleanup next summer...

**Comment by barbequesauce** [comment]
Thanks!

**Pull request merged**